### ISSUE TITLE
Spell check does not work

### ISSUE THREAD

**Issue by gfaidley** [divamgupta/diffusionbee-stable-diffusion-ui#91]
If you misspell a word in the prompt it will be underlined (per default in any text field on Mac OS) but you can not right click on the word to pick the correct spelling.

PS - thanks for the great front end and ease of installation

**Comment by Leland** [comment]
Yeah, context menus on right click are disabled everywhere, and I can't find the culprit yet.

This also means you can't do a copy or paste through the context menu, which some users may prefer to hotkeys.
